tests/schema/tests.py

```diff
@@ -390,6 +390,24 @@ class Meta:
             editor.alter_field(Author, old_field, new_field, strict=True)
         self.assertForeignKeyNotExists(BookWeak, 'author_id', 'schema_author')
 
+    @isolate_apps('schema')
+    def test_model_with_quoted_string_name(self):
+        class Foo(Model):
+            class Meta:
+                app_label = 'schema'
+                db_table = '"foo"'
+
+        with connection.schema_editor() as editor:
+            editor.create_model(Foo)
+        old_field = Foo._meta.get_field('id')
+        new_field = BigAutoField(primary_key=True)
+        new_field.model = Foo
+        new_field.set_attributes_from_name('id')
+        # @isolate_apps() and inner models are needed to have the model
+        # relations populated, otherwise this doesn't act as a regression test.
+        with connection.schema_editor() as editor:
+            editor.alter_field(Foo, old_field, new_field, strict=True)
```

Comment:
We can add `Foo.objects.create()` to ensure that primary key and sequence are still valid.